Sources/XcodeProj/Scheme/XCScheme+PathRunnable.swift

```diff
@@ -3,41 +3,47 @@ import Foundation
 import PathKit
 
 public extension XCScheme {
-    class PathRunnable: Equatable {
+    class PathRunnable: Runnable {
```

Comment:
Making `PathRunnable` subclass `Runnable` means we'll also need to update [ProfileAction](https://github.com/tuist/XcodeProj/blob/main/Sources/XcodeProj/Scheme/XCScheme%2BProfileAction.swift#L35) to support it

Reply:
I overlooked this one. I'll try to add this as well.